### ISSUE TITLE
Is the training code going to be released?

### ISSUE THREAD

**Issue by wenjie-yuan** [YvanYin/Metric3D#11]
I want to train on a multi-dataset for a specific scene, which should improve the accuracy

**Comment by DimplesL** [comment]
Same question, waiting for the code for training, the loss func and the transform method!

**Comment by CaptainEven** [comment]
Looking forward to training code and data processing code

**Comment by zzhmx** [comment]
Same question

**Comment by YvanYin** [comment]
We will release but currently we are still solving the IP issues.

**Comment by FrancescoFornasa** [comment]
Are there any news about this?
